src/pages/projects/forms/NetworkSelectorForm.tsx

```diff
@@ -0,0 +1,49 @@
+import { Select } from "@canonical/react-components";
+import type { SelectProps } from "@canonical/react-components";
+import type { FC } from "react";
+import { useNetworks } from "context/useNetworks";
+
+interface Props {
+  value: string;
+  setValue: (value: string) => void;
+  selectProps?: SelectProps;
+}
+
+const NetworkSelectorForm: FC<Props> = ({ value, setValue, selectProps }) => {
```

Comment:
```suggestion
const NetworkSelector: FC<Props> = ({ value, setValue, selectProps }) => {
```
I think `NetworkSelector` is more aligned with other component names and what this component is intended for.